drone_simulation.py

```diff
@@ -372,14 +447,14 @@ def extra_render(self, env_index: int = 0) -> "List[Geom]":
         geoms: List[Geom] = []
 
         # Agent rotation
-        for agent in self.world.agents:
+        for drone in self.world.agents:
             geoms.append(
-                ScenarioUtils.plot_entity_rotation(agent, env_index, length=0.1)
+                ScenarioUtils.plot_entity_rotation(drone, env_index, length=0.1)
             )
 
         return geoms
 
 if __name__ == "__main__":
     render_interactively(
-        Scenario(), control_two_agents=True, shared_reward=True
+        Scenario(), control_two_drones=True, shared_reward=True
```

Comment:
```suggestion
        Scenario(), control_two_agents=True, shared_reward=True
```
There is no `control_two_drones` parameter in the render_interactively() function